src/type_checker/mod.rs

```diff
@@ -91,12 +91,7 @@ impl TypeChecker {
     }
 
     pub(crate) fn fn_info(&self, qualified: &FullyQualified) -> Option<&FnInfo> {
-        if qualified.module == Some(UserRepo::new("std/builtins")) {
-            // if it's a built-in: get it from a global
-            BUILTIN_FNS.get(&qualified.name)
-        } else {
-            self.functions.get(qualified)
-        }
```

Comment:
I'm puzzled as to why this code existed before, I guess it was useless right?

Reply:
My understanding is it was trying to differentiate the crypto builtin functions and native functions.
But since both the builtins and natives are stored in the same place now, this code seem deprecated.